### DISCUSSION TITLE
I couldn't install this package

### DISCUSSION THREAD

**Discussion by mohsensdpj** [ismaelw/laratex#14] [Q&A]
Hi there,

Unfortunately, I couldn't install and use this package. I'm new to Laravel and would appreciate your help.

I installed texlive on my system (windows 10). the "pdflatex" command works in the command prompt. and everything seems good. but dryRun doesn't work and I can't get any output.
When I run dryRun command as per the documentation, I get this error:

Ismaelw \ LaraTeX \ LaratexException 
(Error in  LaraTex.php line 347.)

**Comment by ismaelw** [comment]
Hi there!

I am not super sure how your setup works. Did you also define a Temp Path inside of your config file?

Kind regards
Ismael

**Comment by mohsensdpj** [comment]
Thanks for your answer.

Actually, I didn't change the default setting in the config file.
this is my Temp Path:         'tempPath' => 'app/',

should I change it?

**Reply by ismaelw** [reply]
Definitely! As I understand, you are on Windows. The default path is for linux systems.

This package uses the temp path to generate the pdf files. I am assuming now that pdflatex cannot find the temp path.

**Comment by mohsensdpj** [comment]
Thanks a lot, for your fast and useful help,

I don't know where is temp path in Windows. I'll search for it. And if I find out where is the temp path in windows, I'll share it here with other users who may use Windows.

**Reply by ismaelw** [reply]
Nevermind. I just went through my code again to see what is going on. The path might only be different for the binpath setting. The temp folder is always inside the laravel storage folder.

Sorry for that.

Another approach: Did you already try to render a pdf from a latex document directly on your system inside a command line prompt? And what is your current configuration for the binPath setting?

**Comment by mohsensdpj** [comment]
Yeah, I tried it. it works and saves a pdf file in the same directory as the .tex file.
I don't change my default binPath configuration. It's:        'binPath' => 'pdflatex',

**Reply by ismaelw** [reply]
And can you also just run pdflatex in your console? Or do you need to enter the path to the pdflatex executable?

**Comment by mohsensdpj** [comment]
yes. there is no need to enter the path to the pdflatex executable.


another point,
I run "php artisan latex:test"
and the test pdf was created in "app/latex/output" path
